src/xdg.c

```diff
@@ -244,6 +245,12 @@ xdg_toplevel_view_maximize(struct view *view, bool maximized)
 	wlr_xdg_toplevel_set_maximized(xdg_toplevel_from_view(view), maximized);
 }
 
+static void
+xdg_toplevel_view_move_to_front(struct view *view)
+{
+	view_impl_move_to_front(view);
+}
+
```

Comment:
Why the extra wrap function here?

Reply:
Have removed - just calling the function in `view-impl-common.c` directly